### PR TITLE
Added Aathil as a contributor

### LINKED ISSUE #1188: New Contributor - Aathil Ahamed
Team: Front End Developers
{
  "title": WordPress Developer,
  "isLead": false,
  "payPerDay": 2800,
  "createdDate": "2020-07-08T02:14:59.307535Z",
  "contributor": {
    "contributorId": U01H0QWK2BG,
    "displayName": Aathil Ahamed,
    "githubUsername": [GITHUB USERNAME],
    "profileImage": https://ca.slack-edge.com/T07PJD1FZ-U01H0QWK2BG-7b8970a84b6c-512,
    "slackUsername": ATLIT
  }
}

### PR DESCRIPTION
**Pull request by Paul-Taiwo**
Fixes #1188 in response to changes requested on [#1201] by @jamessspanggg 

Account number: c6bc0d956d668ed2464684b352226f4df1f86ed213783ba82a97a2fe38252761

### CODE REVIEW

File: src/data/teams.json [hunk 1]
```diff
@@ -641,6 +641,19 @@
...
           "slackUsername": "James Pang"
         }
+      },
+      {
+        "title": "WordPress Developer",
+        "isLead": false,
+        "payPerDay": 2800,
+        "createdDate": "2020-07-08T02:14:59.307535Z",
+        "contributor": {
+          "contributorId": "22",
```
Comment (jamessspanggg) [inline src/data/teams.json:651]: Should be `67` after https://github.com/thenewboston-developers/Website/pull/1202?

**Review by jamessspanggg — CHANGES_REQUESTED** [review]

File: src/data/teams.json (hunk 1 above)
Comment (Paul-Taiwo) [reply src/data/teams.json:651]: Alright.

Comment (jamessspanggg) [reply src/data/teams.json:651]: Please take note that it should be the largest `contributor id + 1` across all teams, not the previous `contributor id + 1`

Comment (Paul-Taiwo) [reply src/data/teams.json]: Noted

**Review by jamessspanggg — APPROVED** [review]

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**